src/SourceBuild/content/Directory.Build.props

```diff
@@ -1,231 +1,200 @@
 <Project>
 
   <PropertyGroup>
-    <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
-    <_SuppressSdkImports>true</_SuppressSdkImports>
+    <_SuppressSdkImports Condition="'$(DotNetBuildFromSource)' == 'true'">true</_SuppressSdkImports>
     <Configuration Condition="$(Configuration) == ''">Release</Configuration>
-    <NetCurrent Condition="'$(SkipArcadeSdkImport)' == 'true'">net9.0</NetCurrent>
   </PropertyGroup>
 
-  <Import Condition="'$(SkipArcadeSdkImport)' != 'true'" Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
-  <PropertyGroup>
-    <!-- Fake, for SDK. -->
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <!-- We have no projects targeting multiple frameworks, so don't include in output path. -->
-    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+  <PropertyGroup Label="CalculateTargetOS">
+    <BuildOS>linux</BuildOS>
```

Comment:
Were you thinking that these would get moved into Arcade?

Reply:
Initially we want to have those defined here in the orchestrator until we get the source build controls implemented. I hope that we can move these properties centrally into the Arcade SDK. If that's not possible (for some build ordering reason), we can keep having them in every individual repository that needs them. I.e. runtime and other repos that cross-compile.